### PR TITLE
dns/unboundbl: reintroduce

### PR DESCRIPTION
**Pull request by mimugmail**
- renamed UnboundBL to Unboundbl, up down up is too error prone
- renamed pluginname to lower only
- added correct license
- fix style

I havent looked at the code yet, but now I dont get a template error when installing.

More tomorrow .. or when merged :)

### CODE REVIEW

File: dns/unboundbl/src/opnsense/mvc/app/views/OPNsense/Unboundbl/index.volt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+
+#}
+
+<div class="alert alert-info hidden" role="alert" id="responseMsg"></div>
+
+<div class="col-md-12"> {{ partial("layout_partials/base_form",['fields':general,'id':'frm_GeneralSettings']) }}
+    <button style="margin: 30px 0px 30px 0px;" class="btn btn-success" id="saveAct" type="button"> <b>{{ lang._('Save') }}</b><i id="saveAct_progress"></i> </button>
+    <p> <b>{{ lang._('REQUIRED') }}:</b> {{ lang._('You must include') }} <span style="font-family: Courier; font-size: 13px; border-radius: 5px 5px 5px 5px; background-color: #000; color: #fff; padding: 5px; margin: 5px;">include:/var/unbound/UnboundBL.conf</span> {{ lang._('in your') }} <a href="../services_unbound.php">Unbound DNS</a> {{ lang._('advanced settings configuration') }}! </p>
```
[the remark this answers is not in the document]
Comment (mimugmail) [reply dns/unboundbl/src/opnsense/mvc/app/views/OPNsense/Unboundbl/index.volt]: You dont need to review yet, I havent looked at the code .. only style and rename :)

File: dns/unboundbl/src/opnsense/mvc/app/models/OPNsense/Unboundbl/ACL/ACL.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<acl>
+   <page-services-unboundbl>
+      <name>Services: UnboundBL</name>
```
[the remark this answers is not in the document]
Comment (mimugmail) [reply dns/unboundbl/src/opnsense/mvc/app/models/OPNsense/Unboundbl/ACL/ACL.xml]: fixed

**Comment by mimugmail** [comment]
@fabianfrz can you merge now? So for those many commits but the syntax was very confusing, upper case, lower case, general, index. For now no errors, values get saved and corretly loaded. The script itself and templating has to be done/fixed by @al-ec .. :)
And please chmod 755 to the script :)

**Comment by mimugmail** [comment]
@fabianfrz can you merge please? :)

**Comment by alectrocute** [comment]
I'll look into template and the script itself tomorrow. Thanks so much, @mimugmail!

**Comment by mimugmail** [comment]
Thx .. but let's take a closer look when @alectrocute touched the scripting stuff. For me it's unclear what he wanted to achieve by all the stuff :) 

Can someone please merge then ..

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by mimugmail** [comment]
@alectrocute you can now work on your scripts in this branch. For me it seems the stuff in template and scripts folder doesn't fit together, rest should be fine.

**Pull request merged**